client/components/QuestionsAndAnswers/QuestionList.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen} from '@testing-library/react';
```

Comment:
@jerry-jma consider adding a space after "screen" or removing the space before "render"